config.yml.example

```diff
@@ -35,6 +35,9 @@ measurement:
   phase-transition-time: 1
   metric-providers:
 
+error:
```

Comment:
When the metric-providers block starts you need to at least fill in one value with the correct indentation. You cannot start from no indentation.

Moving the error block up was the right decision

Reply:
Yeah exactly, realised that!